rumydata/field.py

```diff
@@ -163,8 +175,10 @@ class Empty(Field):
     field in that it will raise errors if any values are found.
     """
 
-    def __init__(self):
+    def __init__(self, all_errors=True, custom_error_msg=None):
         super().__init__(nullable=True)
```

Comment:
Don't rewrite the properties in this `__init__` call, rather pass the values to the `super().__init__()` call